content/tutorials/spatial_wfs_services/index.Rmd

```diff
@@ -498,6 +504,96 @@ knitr::kable(df)
 
 Which indeed corresponds to the data of the coordinate.
 
-This procedure can also be turned into a function with lat, long and properties of interest as parameters.
+This procedure can also be turned into a function with lat, long and properties of interest as parameters:
+
+```{r}
+#' @title Extract soil properties from Flemish soil map
+#'
```

Comment:
Nice function, I've not actually reviewed its code but tested the chunks that use it. It works :tada: !

However I think it would be better to just add this function to `inborutils` and then hyperlink from the tutorial to its online documentation page (and perhaps also to specific lines of its source code: use github's [permalink](https://docs.github.com/en/github/managing-your-work-on-github/creating-a-permanent-link-to-a-code-snippet)), rather than having all the `roxygen2` syntax + R code inside the tutorial. And then just apply `inborutils::soil_map_data_extraction()`, perhaps also showing how to install the latest version of `inborutils`.

Reply:
The benefit of showing the function body is that others can reuse it for other, but similar problems. I agree that eventually the function should go to inborutils.

Reply:
@hansvancalster I'm still a bit in doubt here. I think the benefit of 'direct visibility of the function code' must be weighed against the visual complexity. I think to more casual R users the whole function is rather overwhelming, and from seeing the full function code & documentation they may overlook the purpose & benefit. A possible alternative:

> In order to make your coding work lighter, you can easily put the WFS R code (building the WFS request and retrieving its results) inside a function. Just use [the code](https://github.com/inbo/inborutils/blob/b2592f0eb3ba85b9c50842fb030ec6d8c1d439ee/R/plot_coordinates_on_map.R#L55-L82) of the `soil_map_data_extraction()` function (documented [here](https://inbo.github.io/inborutils/reference/plot_coordinates_on_map.html)) as a template for your own function.
>
> Now this function makes it really easy to request the data:

Reply:
I see your point. Let me add the function to inborutils..

Reply:
See https://github.com/inbo/inborutils/pull/94